_data/navigation.yml

```diff
@@ -347,6 +347,8 @@ reference:
         url: /reference/architecture/load/
       - title: Life of a Bake
         url: /reference/architecture/loab/
+  - title: Pipeline Stages
```

Comment:
Should we maybe put this under a collapsible Pipelines header and call it Stages? I'm sure we can add more pipeline reference material for that header.

Reply:
Such as what? Do we need reference material for triggers?

Reply:
If so, then 👍

Reply:
I decided to do it this way since we don't have other pipeline reference content at the moment, but agreed that as soon as we add more we should create a Pipelines dropdown. Also open to doing it now if you prefer, but my slight preference is to wait until it's actually necessary.

Reply:
I guess it won't require adding more redirects so if we want to defer until more content comes that sounds good to me. 

Triggers, deep dives into stages vs. tasks, synthetic stages, timeouts, SpEL etc... could all be useful to explore.